NoPythonDemo/StyleKitSample/Style.swift

```diff
@@ -70,35 +70,43 @@ class Style: NSObject {
     
     var styleMap = [UIElement:StyleMap]()
     
+    enum StyleKitError: ErrorType {
+        case StyleFileNotFound
+    }
+    
     private override init() {
         super.init()
         serialize(fileName)
     }
-    
-    private func configurationStyleURL(styleFile:String) -> NSURL? {
-        let documentsDirPath = urlForFileInDocumentsDirectory(styleFile)
-        if NSFileManager.defaultManager().fileExistsAtPath(documentsDirPath.path!)   {
-            return documentsDirPath
-        }
-        let bundlePath = NSBundle.mainBundle().pathForResource(styleFile, ofType: nil)
-        return NSURL(fileURLWithPath: bundlePath!)
-    }
-    
-    private func urlForFileInDocumentsDirectory(fileName: String) -> NSURL {
-        let docsDir = NSFileManager.defaultManager().URLsForDirectory(.DocumentDirectory, inDomains: .UserDomainMask)[0]
-        return docsDir.URLByAppendingPathComponent(fileName)!
-    }
-    
+
     private func checkIfImageExist(name:String) -> Bool {
         return UIImage(named: name) == nil ? false : true
     }
     
-    private func serialize(styleFile:String) {
-        
-        let stylePath = configurationStyleURL(styleFile)!
-        
+    private func getStylePath() throws -> NSURL {
+        guard let string = NSBundle.mainBundle().infoDictionary?["StyleKit-DesignatedFolder"] as? String,
```

Comment:
Really think this should use 'if let' logic instead of 'guard let'. Conventionally 'guard let' is reserved for the case where you expect that something always to be true.

In the case of the documents dir, guard let makes perfect sense.